src/test/java/software/amazon/smithy/lsp/SmithyInterfaceTest.java

```diff
@@ -105,10 +105,16 @@ public void runValidators() throws Exception {
         assertTrue(result.isRight());
         List<ValidationEvent> validationEvents = result.getRight().getValidationEvents();
         assertFalse(validationEvents.isEmpty());
-        assertEquals(
-            "Proto index 1 is used muliple times in members name,age of shape (structure: `some.test#MyStruct`).",
-                validationEvents.get(0).getMessage()
-        );
+
+        String expectedMessage = "Proto index 1 is used muliple times in members name," +
```

Comment:
Nit, there is a typo on "muliple" and a missing white space after the comma.

Reply:
Yea, that's just the message the validator produces, which lives in a jar in test resources. I don't remember where exactly it came from